js/login.js

```diff
@@ -0,0 +1,9 @@
+$(document).ready(function(){
```

Comment:
Inconsistent opening {, would prefer like "function() {" below.

Also, in jQuery using "$(function() {" is sufficient for creating a document ready event.